freetextresponse/freetextresponse.py

```diff
@@ -264,6 +265,7 @@ def student_view(self, context={}):
                 'problem_progress': self._get_problem_progress(),
                 'prompt': self.prompt,
                 'student_answer': self.student_answer,
+                'before_due': not self.is_past_due(),
```

Comment:
I'd prefer to use the term `past_due` consistently throughout.

Reply:
I actually specifically made this `before_due` to make the django template more readable:

```django
{% if before_due %}
    # Stuff
{% endif %}
```

vs

```django
{% if past_due != None%}
    # Stuff
{% endif %}
```